.mergify.yml

```diff
@@ -88,6 +84,8 @@ pull_request_rules:
         assignees: &BackportAssignee
           - "{{ merged_by|replace('mergify[bot]', label|select('equalto', 'community')|first|default(author)|replace('community', '@solana-labs/community-pr-s
 ubscribers')) }}"
+        title:
+          - "{{ destination_branch }}: {{ title }} (backport of #{{ number }})"
```

Comment:
also added "of" after "backport", because I'm confused on every backport if the PR number in parenthesis in the default title is the backport or original PR and tend to get it wrong ~50% of the time